commands/bootstrap/__tests__/bootstrap-command.test.js

```diff
@@ -138,6 +138,17 @@ describe("BootstrapCommand", () => {
     });
   });
 
+  describe("with pnpm", () => {
+    it("should throw validation error", async () => {
+      const testDir = await initFixture("pnpm");
+      const command = lernaBootstrap(testDir)();
+
+      await expect(command).rejects.toThrow(
+        "Bootstrapping with pnpm is not supported. Use pnpm directly to manage dependencies (https://pnpm.io/cli/install)."
```

Comment:
Please align to match formatting of error for `add`, e.g. when it comes to the URL